torchtext/vocab.py

```diff
@@ -30,11 +30,9 @@ class Vocab(object):
         itos: A list of token strings indexed by their numerical identifiers.
     """
 
-    # TODO (@mttk): Populate classs with default values of special symbols
-    UNK = '<unk>'
```

Comment:
I think this will break the text_classification dataset here.
https://github.com/pytorch/text/blob/9afcfcdfdcb668780aba59fa1d06411645d54dd7/torchtext/datasets/text_classification.py#L52

Reply:
I changed that line already :)